testhelpers/docker_registry.go

```diff
@@ -182,6 +198,13 @@ func (r *DockerRegistry) EncodedLabeledAuth() string {
 	return base64.StdEncoding.EncodeToString([]byte(fmt.Sprintf(`{"username":"%s","password":"%s"}`, r.username, r.password)))
 }
 
+// Add stores the given key name with the provided ImagePrivileges. For example
+// Add("my-image", NewImagePrivileges("foo-writable-readable")) will save "my-image" as a
+// readable and writable image into the registry
```

Comment:
```suggestion
// readable and writable image into the registry.
// It must be called before Start to have any effect.
```

Reply:
Fixed in the following [commit](https://github.com/buildpacks/imgutil/pull/126/commits/1a4373ac85d225b6bd3771c3ccadae2a239cf6db)